jimp/index.d.ts

```diff
@@ -0,0 +1,163 @@
+// Type definitions for jimp 0.2
+// Project: https://github.com/oliver-moran/jimp#readme
+// Definitions by: Jack Works <https://github.com/Jack-Works>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+declare namespace jimp {
+	type Callback<T> = (this: JimpImage, err: Error, data: T) => void;
+	type ResizeMode = string;
+	type PresetFont = string;
+	type FileMINE = string;
+	type AlignMode = number;
+	type FilterType = number;
+	type Font = any | PresetFont;
+
+
+	class JimpImage {
+		constructor(width: number, height: number, callback?: Callback<JimpImage>);
+		constructor(width: number, height: number, initialColor: number, callback?: Callback<JimpImage>);
```

Comment:
Combine the two, use
`constructor(width: number, height: number, initialColor?: number, callback?: Callback<JimpImage>)`